openstack/cbshelf/templates/backend-configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ .Release.Namespace }}
```

Comment:
As a general remark that applies to all template files, you do not need to set `metadata.namespace = {{.Release.Namespace}}`. That's automatically implied by Helm.